src/Datasource/EntityTrait.php

```diff
@@ -796,7 +796,7 @@ public function getDirty()
     public function &getOrFail($property)
     {
         $value = $this->get($property);
-        if (!$this->propertyExists($property) && $value === null) {
+        if ($value === null && !$this->propertyExists($property)) {
             throw new \InvalidArgumentException('Property "' . $property . '" not found.');
```

Comment:
`\InvalidArgumentException` <= Should IMHO be a use statement at the top then call it without the `\`

Reply:
Just include my sniffer :) To auto-fix those issues. Problem solved more cleanly.
But sure, one can also manually adjust all those in the code base.

Reply:
You are right. I just saw and changed that. But in comments I should still use the absolute path, right?

Reply:
Yes, comments are always full FQCN.

Reply:
You couldnt, but CakePHP community could have :)